gstreamer-gl-sys/src/lib.rs

```diff
@@ -519,6 +519,22 @@ impl ::std::fmt::Debug for GstGLDisplayClass {
     }
 }
 
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct GstGLDisplayEGLClass {
```

Comment:
This needs to be behind feature flags. EGL is not always available unfortunately